apps/mobile-wallet/src/components/inputs/Input.tsx

```diff
@@ -81,6 +90,15 @@ const Input = <T extends InputValue>({
     fontSize: withSpring(!isActive ? 15 : 11, fastSpringConfiguration)
   }))
 
+  useEffect(() => {
+    const fetchCopiedText = async () => {
+      const text = await getStringAsync()
+      setCopiedText(text)
+    }
+
+    fetchCopiedText()
+  })
```

Comment:
The dependency array is missing.

Also, I started preferring this pattern, WDYT?

```suggestion
useEffect(() => {
    getStringAsync().then(setCopiedText)
  }, [])
```

Reply:
I also like it better I think.

Reply:
Changed to the pattern you proposed